spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:item) { 'kimchi' }
+  let(:quantity) { 2 }
+  let(:menu) { double(:menu, dishes: { "kimchi" => 2 }) }
+
+
+  context '#initialize' do
+    it 'has an empty basket' do
+      expect(order.basket).to be_empty
+    end
+  end
+
+  context '#add_basket' do
+    it 'adds items to basket' do
+      expect(order.add_basket(item, quantity)).to change{order.basket.length}.by(1)
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)